WordPress/gutenberg
# Use a DropdownMenu for menu selection in the navigation screen

## Description
Closes #25312.

Use a DropdownMenu component for choosing a menu.

Needs accessibility feedback ...

## How has this been tested?
1. Visit the navigation screen
2. Make sure you have more than one menu
3. Try selecting menus




## Screenshots 
<img width="846" alt="Screenshot 2020-09-17 at 12 37 06 pm" src="https://user-images.githubusercontent.com/677833/93421021-e070af00-f8e2-11ea-80c8-d94a3fdcc257.png">

## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
